src/baseai.ts

```diff
@@ -0,0 +1,54 @@
+import Ain from "@ainblockchain/ain-js";
```

Comment:
minor) baseAi.ts following other files' patterns (e.g. factoryBase.ts)?

Reply:
Done!

Reply:
sorry, I can't see any updates.

Reply:
sorry, I didn't know git doesn't track changes in capital letters in file names..
I found a [issue](https://stackoverflow.com/questions/10523849/how-do-you-change-the-capitalization-of-filenames-in-git) and took it again!